README.md

```diff
@@ -19,6 +19,7 @@ Dataset name | Organs | Staining | Link | Size | Data | Task | WSI/Patch | Other
 --- | --- | --- | --- |--- |--- |--- |--- |--- |---
 ACDC-LungHP [1a], [1b] | Lung | H&E | [data](https://acdc-lunghp.grand-challenge.org/), [paper](https://ieeexplore.ieee.org/document/9265237) | Train: 150, Test: 50 | images + xml | seg + classi | wsi | | 2019
 ACROBAT 2022 [66] | Breast | Multiple (IHC, H&E) | [data](https://acrobat.grand-challenge.org/), [paper](https://openreview.net/pdf?id=5TTocO2VI3r) | Train: 750 train; Valid: 100; Test: 300 | images (1 H&E match to 1-4 IHC) + landmarks | registration | wsi | 40x - Hamamatsu | 2022
+Adipocyte [94] | skin | H&E | [data](https://github.com/ieee8023/countception), [paper](https://arxiv.org/abs/1703.08710) | 150 patches | images+mask |  cell detection | patch (600x600) | 40x | 2017
```

Comment:
it is `200 patches` and `120X150`.

Reply:
sorry for my fault, I have corrected it.